### PR TITLE
set environment variables inside created container #149

### PR DESCRIPTION
**Pull request by igormusinov**
When you send get request like "http://127.0.0.1:8000/hub/oauth_login?repourl=https://github.com/everware/everware-dimuon-example&api_token=`value`"

The API_TOKEN variable in the container will have the `value`

### CODE REVIEW

**Comment by anaderi** [comment]
This PR is not exactly what is needed. it takes variables with specific name `api_token`, and sets it into newly created environment. but idea was that user can pass set of variables as PARAM1=VALUE1&PARAMX=VALUEY etc. 
in order to do so, you have to read all variables, create dictionary of them, and add this dictionary to options passed to spawn in the end.

**Comment by igormusinov** [comment]
Okey, it is the next step.

С уважением,
Мусинов Игорь

2016-10-15 12:39 GMT+03:00 Andrey Ustyuzhanin notifications@github.com:

**Comment by anaderi** [comment]
Ahem... ?

**Comment by igormusinov** [comment]
Now, all url's parameteres  are available as environment variables.
Can check:
!echo $<name of parameter>

File: everware/spawner.py [hunk 1]
```diff
@@ -133,17 +133,22 @@ def _options_form_default(self):
...
 
     @property
     def form_repo_url(self):
         """Repository URL as submitted by the user."""
         return self.user_options.get('repo_url', '')
+        
+    @property
+    def form_api_token(self):
+        """Api_token for user."""
+        return self.user_options.get('api_token', '')
```
Comment (anaderi) [inline everware/spawner.py]: Why do you need this function?

File: everware/authenticator.py [hunk 2]
```diff
@@ -107,15 +106,17 @@ def get(self):
                 'oauth_callback'
             )
         )
-        
+
         redirect_uri = self.authenticator.oauth_callback_url or guess_uri
         self.log.info('oauth redirect: %r', redirect_uri)
 
         repourl = self.get_argument('repourl', '')
+        api_token = self.get_argument('api_token', '')
```
Comment (anaderi) [inline everware/authenticator.py:114]: why do you need this line?

**Review by anaderi — COMMENTED** [review]
look inline

**Pull request merged**